netbout-client/src/test/java/com/netbout/client/RtAttachmentsITCase.java

```diff
@@ -85,4 +86,50 @@ public void postsAndReads() throws Exception {
         attachments.delete(name);
     }
 
+    /**
+     * RtAttachments can obtain author of an attachment.
+     * @throws Exception If there is some problem inside
+     */
+    @Test
+    public void obtainsAuthor() throws Exception {
+        final User user = NbRule.get();
+        final Alias alias = user.aliases().iterate().iterator().next();
+        final Inbox inbox = alias.inbox();
+        final Bout bout = inbox.bout(inbox.start());
+        final Attachments attachments = bout.attachments();
+        final String name = "name";
+        attachments.create(name);
+        final Attachment attachment = attachments.get(name);
+        MatcherAssert.assertThat(
+            attachment.author(), Matchers.equalTo(alias.name())
+        );
+        attachments.delete(name);
+    }
+
+    /**
+     * RtAttachments can obtain the creation date of an attachment.
+     * @throws Exception If there is some problem inside
+     */
+    @Ignore
```

Comment:
@mkordas it is also related to the puzzle I created. We can't read date with DyAttachment yet, so we can't send it in XeAttachment and read it with RtAttachment. That is why it is ignored.